style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -273,6 +273,11 @@ const metadata: StyleDictionary.MetadataIndex = {
     public: true,
     themeable: true,
   },
+  colorBorderDividerDimmed: {
```

Comment:
Those existing tokens are private (not public tokens), so it's a bit different because:
1. They can't be themed, so using a appearance-specific term like "dimmed" or a color name is OK
2. It doesn't matter that much in general, because we can change this name at any time without affecting anyone.

Once a token is public, then their name becomes part of a public API. So it's a good time to rethink its name :)

Reply:
Had offline discussion, and we decided to go with `colorBorderDividerSecondary`. This way the color token can be more scalable for the future. Now this token is used for the row dividers for table and collection preferences as horizontal dividers, but it could be used for vertical dividers in the future.